src/vellum/workflows/nodes/displayable/tool_calling_node/utils.py

```diff
@@ -520,12 +520,25 @@ def create_function_node(
             },
         )
     else:
-        # For regular functions, use FunctionNode
+
+        def create_function_wrapper(func):
+            def wrapper(self, **kwargs):
+                merged_kwargs = kwargs.copy()
+                inputs = getattr(func, "_inputs", {})
```

Comment:
[nit] let's use dunder's and namespace it (`__vellum_inputs__`). It's a more standard python convention for hacking onto classes, and the namespace allow us to play nicely with other libs